### PR TITLE
Signin signup v1

### PR DESCRIPTION
**Pull request by kubaczechowski**

### CODE REVIEW

File: src/stores/UserStore.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      else return "";
+    },
+  },
+  actions: {
+  /*  createUserandSignUp(username: string, password: string) {
+      userService
+        .createUserandSignUp(username, password)
+        .then((user) => (this.loggedInUser = user))
+        .catch((err) => console.log(err));
+    }, */
```
Comment (kamila0801) [inline src/stores/UserStore.ts]: if we don't need that maybe let's delete?

File: src/views/Signup.vue [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+async function signIn(this: any){
+  if(!username.value){
+    showMessage.value= true;
+    messageIfSth.value = "empty username .."
+  }else{
+    if(signingUp.value ==="Sign in")
+        messageIfSth.value = "Username doesn't exists or pass is wrong";
+    else
+        messageIfSth.value = "Username already exists";
```
Comment (kamila0801) [inline src/views/Signup.vue:86]: great that you covered those cases!

File: src/App.vue [hunk 3]
```diff
@@ -7,7 +7,7 @@ provide("userService", new UserService());
 </script>
 
 <template>
-  <RouterLink to="/users">Users</RouterLink>
+<!--  <RouterLink to="/users">Users</RouterLink>-->
```
Comment (kamila0801) [inline src/App.vue]: remember to out-comment it when merging, to hide the navbar when signIn/signUp you can add v-if store.loggedUser and then it won't be visible if the user isn't logged in :)

File: src/router/index.ts [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
 const router = createRouter({
   history: createWebHistory(import.meta.env.BASE_URL),
   routes: [
     {
-      path: '/',
-      name: 'home',
-      component: HomeView
+      path: '/signin',
+      name: 'signin',
+      component: Signup
```
Comment (kamila0801) [inline src/router/index.ts:11]: maybe we could set signIn path as default when opening the website? at least if user isn't loggedIn

File: src/stores/UserStore.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    signInUser(signedInUser: GetUsersDto){
+        this.loggedInUser =signedInUser;
+    }
+
```
Comment (kamila0801) [inline src/stores/UserStore.ts:22]: I don't think you should use this dto, even tho it works it has other purpose (it has a field 'status' which says if that user is our friend etc)

File: src/views/Signup.vue [hunk 6]
```diff
@@ -0,0 +1,153 @@
+<template>
+  <div class="min-h-screen flex items-center justify-center">
+
+    <!--login box    -->
+    <div class="px-16 shadow-2xl pt-16 pb-11 rounded-2xl  w-1/3 bg-amber-100">
+
+      <h3  class="text-slate-900 font-bold text-5xl mb-12 text-center	">{{signingUp}}</h3>
+
+      <form @submit.prevent="signIn" class="text-slate-900 space-y-7 py-2 px-4">
```
Comment (kamila0801) [inline src/views/Signup.vue:9]: looks great, bold color choice hahaha

**Review by kamila0801 — COMMENTED** [review]
great job 💯

File: src/stores/UserStore.ts (hunk 1 above)
Comment (kubaczechowski) [reply src/stores/UserStore.ts]: Sure!

**Pull request merged**